docs/webhooks/index.html

```diff
@@ -328,7 +449,127 @@ <h3 id="companyTestSessionFinished">companyTestSessionFinished</h3>
   }
 };</code></pre>
                 </div>
-		<h2>Events for CodeSignal Certify</h2>    
+<div class="subsection">
+   <h3 id="auditorworkflow">CodeSignal Auditor workflow</h3><br>
+       <img alt="Diagram of webhooks triggered on CodeSignal auditor flow" width="100%" src="../images/auditor-events.png" />
+</div>
+
+                <div class="subsection">
+                    <h4 id="preScreenResultNotVerified">preScreenResultNotVerified</h4>
+                    <p>
+                        Fired when an auditor rejects a proctored test session.
+                    </p>
+                    <div>Request format:</div>
+                    <pre><code>{
+  event: 'preScreenResultNotVerified',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    rejectedReasons: string[],
+  }
+};</code></pre>
+                    <div>Sample request:</div>
+                    <pre><code>{
+  event: 'preScreenResultNotVerified',
+  triggeredOn: 1553720789347,
+  payload: {
+    testSessionId: 'lehu382hdleh29',
+    testId: 'dhey29hcl28hl28',
+    testTitle: 'Software Engineer intern',
+    candidateEmail: 'john.doe@gmail.com',
+    candidateName: 'John Doe',
+    rejectedReasons: ['Two humans in the room', 'Screen not visible]
```

Comment:
I think it would be ideal to use examples of our "real" rejection reasons here if possible. I think "Screen not visible" is realistic but I don't believe "Two humans in the room" is exactly the phrasing we use on prod?